Nosach/web-quiz-project/pom.xml

```diff
@@ -54,7 +54,6 @@
 		<dependency>
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-test</artifactId>
-			<scope>test</scope>
```

Comment:
Why you deleted this from pom?

Reply:
At first I couldn't start tests so I tried this. Then forgot to get back.